examples/medplum-mso-demo/src/utils/enrollment.ts

```diff
@@ -0,0 +1,267 @@
+ * @param patient - The patient to unenroll.
+ * @param organization - The organization to unenroll the patient from.
+ */
+export async function unEnrollPatient(
+  medplum: MedplumClient,
+  patient: Patient,
+  organization: Organization
+): Promise<void> {
+  const accounts = patient.meta?.accounts || [];
+  const orgReference = getReferenceString(organization);
+
+  // Create parameters with all accounts except the one to remove
+  const parameters = {
+    resourceType: 'Parameters',
+    parameter: accounts
+      .filter((a: Reference) => a.reference !== orgReference)
+      .map((account) => ({
+        name: 'accounts',
+        valueReference: {
+          reference: account.reference,
+        },
+      })),
+  };
+
+  //NOTE: If the patient has ben unenrolled from all organizations, this is a workaround to conform to the FHIR spec's
+  //inability to handle empty arrays. Instead of sending an empty array, we send the patient's own reference.
+  if (parameters.parameter.length === 0) {
+    parameters.parameter = [
+      {
+        name: 'accounts',
+        valueReference: createReference(patient),
+      },
+    ];
+  }
+
+  try {
+    await medplum.post(`fhir/R4/Patient/${patient.id}/$set-accounts`, parameters);
+  } catch (error) {
+    throw new Error(normalizeErrorString(error));
+  }
+}
+
+/**
+ * Unenrolls a practitioner from an organization.
+ * @param medplum - The Medplum client.
+ * @param practitioner - The practitioner to unenroll.
+ * @param organization - The organization to unenroll the practitioner from.
+ */
+export async function unEnrollPractitioner(
+  medplum: MedplumClient,
+  practitioner: Practitioner,
+  organization: Organization
+): Promise<void> {
+  const projectMembershipSearch = await medplum.search('ProjectMembership', {
+    profile: getReferenceString(practitioner),
+  });
+
+  const membershipResource = projectMembershipSearch.entry?.[0]?.resource;
+
+  if (membershipResource) {
+    //remove organization from access
+    membershipResource.access = membershipResource.access?.filter((access) =>
+      access.parameter?.some(
+        (param: ProjectMembershipAccessParameter) =>
+          param.valueReference?.reference !== getReferenceString(organization)
+      )
+    );
+
+    try {
+      await medplum.updateResource(membershipResource);
+    } catch (error) {
+      throw new Error(normalizeErrorString(error));
+    }
+  }
+}
+
+/**
+ * Gets practitioners enrolled in a specific organization.
+ * @param medplum - The Medplum client.
+ * @param organization - The organization to get enrolled practitioners for.
+ * @returns Array of practitioners enrolled in the organization.
+ */
+export async function getEnrolledPractitioners(
+  medplum: MedplumClient,
+  organization: Organization
+): Promise<Practitioner[]> {
+  // Search ProjectMemberships first
+  const membershipSearch = await medplum.search('ProjectMembership', {
+    _include: 'ProjectMembership:profile',
```

Comment:
This `_include` seems unnecessary, since you're making a separate search for the practitioners below